packages/jimp/README.md

```diff
@@ -80,43 +80,65 @@ If you're using webpack you can set `process.browser` to true and your build of
 
 ## Basic usage
 
-The static `Jimp.read` method takes the path to a file, URL, dimensions, a Jimp instance or a buffer and returns a Promise:
```

Comment:
This makes sense to still be here. Dimensions refers to height and width of an image. You can do the following to make an image

```js
const image = await Jimp.read(200, 400) // 200x400 image with blank background
```

Reply:
Didn't know about this. Fixed.